classes/phing/types/DirSet.php

```diff
@@ -46,9 +46,9 @@ public function __construct($dirset = null)
     public function __clone()
     {
         if ($this->isReference()) {
-            return new DirSet($this->getRef($this->getProject()));
+            new DirSet($this->getRef($this->getProject()));
         } else {
-            return new DirSet($this);
+            new DirSet($this);
```

Comment:
Hm does this actually do anything? I'm not sure we even need to keep the `__clone` methods to be honest, what do you think?